### PR TITLE
Add qibo

### PR DESCRIPTION
**Pull request by scarrazza**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by scarrazza** [comment]
@scopatz is there some way to change `channel_priority` to `flexible` from the recipe directory? I am modifying the `.scripts/build_steps.sh` because otherwise I cannot install TensorFlow 2.1.

**Comment by scopatz** [comment]
No not really. We require all of our packages to be on conda-forge or defaults.  The best way would be to get tensorflow on conda-forge, which I know is very difficult

**Comment by scarrazza** [comment]
@scopatz tensorflow 2.2 for linux is already available from the defaults (https://anaconda.org/anaconda/tensorflow), see https://github.com/conda-forge/tensorflow-feedstock/pull/104. Is there some way to use that?

**Comment by scopatz** [comment]
yeah, that should be picked up then, depending on the pinnings here

**Comment by scarrazza** [comment]
@scopatz just reverted the change in `.scripts/build_steps.sh`, however the build fails because the `tensorflow >2.1` constrain is not satisfied (even if this package is in defaults). Note the message `Note that strict channel priority may have removed packages required for satisfiability.`, see [this line](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=203156&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0&l=599).

File: recipes/qibo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  build:
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - pip
+    - tensorflow >2.1
+  run:
+    - python
+    - joblib
+    - matplotlib
```
Comment (scopatz) [inline recipes/qibo/meta.yaml:29]: this really does need to be `matplotlib-base`

**Comment by scopatz** [comment]
CC @conda-forge/staged-recipes

File: recipes/qibo/meta.yaml (hunk 1 above)
Comment (scarrazza) [reply recipes/qibo/meta.yaml]: Thanks.

**Comment by jakirkham** [comment]
This didn't get reviewed. So removing stale

**Comment by jakirkham** [comment]
cc @conda-forge/staged-recipes

**Pull request closed without merging**